src/main/java/baritone/utils/BlockStateInterface.java

```diff
@@ -97,9 +97,9 @@ public BlockState get0(BlockPos pos) {
     }
 
     public BlockState get0(int x, int y, int z) { // Mickey resigned
-        y -= worldData.dimension.minY();
+        y -= world.dimensionType().minY();
```

Comment:
does JIT optimize these function calls away, or should I move this to a `private final int`

dimensionType just returns a final field
minY is also just returning a final field

Reply:
can you explain how you fixed this in low IQ speech. I appreciate it

Reply:
world -> worldData as world can sometimes be null

Reply:
![Capture](https://user-images.githubusercontent.com/67770237/174524083-87fcaae8-4cb7-4a12-9c26-2ee1ff3612ba.PNG)
checking through but I cant find a file for worldData

Reply:
:rock: :brain: